src/App.tsx

```diff
@@ -40,6 +40,7 @@ export class App extends React.Component<{}, {
       .catch ((err): void => {
```

Comment:
`initializeArc` doesn't throw exceptions.  It returns false if it fails.  Note the `while` loop.